### PR TITLE
docs: improve quick start

### PR DESCRIPTION
**Pull request by robert-zaremba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,56 +32,26 @@ To attract more web developers we plan to build a JavaScript DSL for TEAL with T
...
+Make sure that the node you are connecting to has a `"EnableDeveloperAPI": true` option set in the `<node_data>/config.json`. This is required to compile smart contracts using REST / SDK.
 
-### Algorand Node requirements
+Follow our [infrastructure guide](https://github.com/scale-it/algo-builder/tree/master/infrastructure/README.md) for more details.
 
-- algod v2.1.6-stable or higher
+### Connecting to an external Algorand Node
 
-Make sure that the node you are connecting to has a `"EnableDeveloperAPI": true` option set in the `<node_data>/config.json`. This is required to compile smart contracts using REST / SDK.
+Instead of using a local node, you can connect to an external one by setting the network configuration in the [`algob.config.js`](guide/algob-config) file. More about the project setup and config file is described in the Quick Start section below.
```
Comment (vuvth) [inline README.md:50]: Should change to guide/algob-config to https://github.com/scale-it/algo-builder/blob/master/docs/guide/algob-config.md

File: README.md [hunk 2]
```diff
@@ -129,103 +101,153 @@ Otherwise you can use a system/user-wide `pyteal` installation:
...
-1.  Add deployment scripts in `scripts` directory.
-1.  Run `yarn run algob deploy` to compile and deploy everything.
-1.  Run `yarn run algob run scriptPath/scriptName` to run script.
-1.  To run `algob` on different network (by default the `default` network is used) use
+1. You don't need to install and run Algorand Node yourself. You can connect to any network provider (eg Pure Stake).
+1. Add assets and smart-contracts in the `assets` directory.
+1. Add deployment scripts in `scripts` directory.
+1. Run `yarn run algob deploy` to compile and deploy everything (all scripts nested directly in /scripts).
+1. Run `yarn run algob run scriptPath/scriptName` to run a script.
+1. To run `algob` on different network (by default the `default` network is used) use
```
Comment (ratik21) [inline README.md:117]: Are these are pointed as `1.`?

File: README.md [hunk 3]
```diff
@@ -129,103 +101,153 @@ Otherwise you can use a system/user-wide `pyteal` installation:
...
+Then you can add `"algob": "algob"` into the package.json scripts section and use it with `yarn run algob`.
 
-Using the `algob unbox-template` command, the developers can get a pre-built dApp project containing scripts to deploy assets and smart contracts with react.js interactive frontend. The templates use [AlgoSigner](https://github.com/PureStake/algosigner) to securely sign and send transactions to an Algorand Blockchain Network.
+Finally you need to create [`algob.config.js`](docs/algob-config) file.
 
-Detailed description about the templates can be found [here](https://github.com/scale-it/algo-builder-templates#algo-builder-templates).
+### Installation from master
 
-# Contributing
+We recommend cloning our GitHub _master_ branch if you want to play with [templates](https://github.com/scale-it/algo-builder-templates) and [examples](https://github.com/scale-it/algo-builder/tree/master/examples) (all our example smart contracts are tested against _master_). The master branch corresponds to the latest released version. All examples are linked to the
```
Comment (ratik21) [inline README.md:146]: `All examples are linked to the `?

**Review by ratik21 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vuvth) [reply README.md:50]: Oh, Thanks!

File: README.md (hunk 3 above)
Comment (robert-zaremba) [reply README.md:146]: thanks, forgot to finish the sentence.

**Pull request merged**